src/main/java/com/yahoo/sketches/quantiles/DoublesSketch.java

```diff
@@ -138,7 +138,7 @@
    * received in exactly the same order. This is only useful when performing test comparisons,
    * otherwise is not recommended.
    */
-  public static final Random rand = new Random(1);
+  public static Random rand = new Random();
```

Comment:
i thought we were avoiding this since we didn't want people to break their own quantiles sketches